### PR TITLE
New version: Stoplight.Prism version 5.5.4

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137673)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2024-02-02 18:43:49.489 [CLI ] ShellExecute installer failed: 1

(Automated response - build 695.)

File: manifests/s/Stoplight/Prism/5.5.4/Stoplight.Prism.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Stoplight.Prism
+PackageVersion: 5.5.4
+InstallerType: exe
```
Comment (stephengillie) [inline manifests/s/Stoplight/Prism/5.5.4/Stoplight.Prism.installer.yaml:6]:
```suggestion
InstallerType: portable
```

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Duplicate of https://github.com/microsoft/winget-pkgs/pull/137661

**Pull request closed without merging**